packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -943,45 +943,29 @@ class RenderClipRect extends _RenderCustomClip<Rect> {
 
 /// Clips its child using a rounded rectangle.
 ///
-/// Creates a rounded rectangle from its layout dimensions and the given x and
-/// y radius values and prevents its child from painting outside that rounded
+/// Creates a rounded rectangle from its layout dimensions and the given border
+/// radiu and prevents its child from painting outside that rounded
 /// rectangle.
 class RenderClipRRect extends RenderProxyBox {
   /// Creates a rounded-rectangular clip.
   RenderClipRRect({
     RenderBox child,
-    double xRadius,
-    double yRadius
-  }) : _xRadius = xRadius, _yRadius = yRadius, super(child) {
-    assert(_xRadius != null);
-    assert(_yRadius != null);
+    BorderRadius borderRadius
```

Comment:
either way, mention this argument in the dartdoc for the constructor, like "The border radius is required" or "the border radius defaults to a rectangle with right-angled corners" or something